db/migrate/20180321094200_populate_full_approval_workflow.rb

```diff
@@ -0,0 +1,13 @@
+class PopulateFullApprovalWorkflow < ActiveRecord::Migration[5.0]
+  def up
+    Case::Base.find_each do |kase|
```

Comment:
We should make this data migration safe against future DB and system changes, as per suggestions here: https://railsguides.net/change-data-in-migrations-like-a-boss/

- define a `Case::Base` class in here (or call it whatever you like) inheriting from `ActiveRecord::Base`
- copy-paste the method `flagged_for_press_office_clearance?`, and whatever associations or methods it uses, into here, OR use our own test for whether the case has been flagged for clearance by press

Reply:
I think I'd rather spend time making the db:reseed and reset rake tasks load the schema from structure.sql rather than running all the migrations each time, in which case this isn't an issue.  What do  you think?

Reply:
I wouldn't mind making the rake tasks reload the db structure, but shouldn't we do both? The migrations to ensure that we don't have problems later (which has already happened) and to adhere to the [Principle of Least Astonishment](https://en.wikipedia.org/wiki/Principle_of_least_astonishment).

It should be pretty quick to do, would this be enough?

```
class Team < ActiveRecord::Base
end

class BusinessUnit < Team
  def self.press_office
    find_by!(code: Settings.press_office_team_code)
  end
end

class Case < ActiveRecord::Base
  def flagged_for_press_office_clearance?
    approving_teams.include?(BusinessUnit.press_office)
  end
end
```